src/components/DownloadComponents/NextLessonCard/NextLessonCard.test.tsx

```diff
@@ -0,0 +1,57 @@
+import NextLessonCard from "./NextLessonCard";
+
+import renderWithTheme from "@/__tests__/__helpers__/renderWithTheme";
+
+describe("NextLessonCard", () => {
+  it("should render component", () => {
+    const { getByText } = renderWithTheme(
+      <NextLessonCard
+        lessonTitle="test-lesson"
+        programmeSlug="test-programme"
+        unitSlug="test-unit"
+        lessonSlug="test-slug"
+      />,
+    );
+
+    const nextLessonCardTitle = getByText("test-lesson");
+
+    expect(nextLessonCardTitle).toBeInTheDocument();
+  });
+  it("should render see lesson link with correct href", () => {
+    const { getByTestId } = renderWithTheme(
+      <NextLessonCard
+        lessonTitle="test-lesson"
+        programmeSlug="test-programme"
+        unitSlug="test-unit"
+        lessonSlug="test-slug"
+      />,
+    );
+
+    const seeLessonLink = getByTestId("see-lesson-link");
```

Comment:
can we `getByRole` instead of using testId?

Reply:
Amended ✅